docs/administrative/change_account_name.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Some of the images like `Account Product & Severity...` and `Create Ticket.png` etc. are being added in this PR, but are they actually being used? I don't see them linked below. If they are no longer being used then can we remove them?